docs-conceptual/azps-3.5.0/install-az-ps.md

```diff
@@ -130,54 +142,67 @@ Connect-AzAccount
 
 ## Update the Azure PowerShell module
 
-If you originally used Install-Module, then you should use [Update-Module](/powershell/module/powershellget/update-module) to get the latest version. If you originally used the MSI package then you should download and install the new MSI in order to udpate. If you have any issues with updating using the package from PowershellGet then you will need to __reinstall__, rather than
-just __update__. This is done in the same way as installing, but you may need to add the `-Force` argument:
+To update any PowerShell module, you should use the same method used to install the module. For
+example, if you originally used `Install-Module`, then you should use
+[Update-Module](/powershell/module/powershellget/update-module) to get the latest version. If you
+originally used the MSI package then you should download and install the new MSI package.
+
+The PowerShellGet cmdlets cannot update modules that were installed from an MSI package. MSI
+packages do not update modules that were installed using PowerShellGet. If you have any issues
+updating using PowershellGet then you should **reinstall**, rather than **update**. Reinstalling is
+done the same way as installing, but you need to add the `-Force` parameter:
 
 ```powershell
 Install-Module -Name Az -AllowClobber -Force
 ```
 
-Although this can overwrite installed modules, you may still have older versions left on your system.
-To learn how to remove old versions of Azure PowerShell from your system, see [Uninstall the Azure PowerShell module](uninstall-az-ps.md).
+Unlike MSI-based installations, installing or updating using PowerShellGet does not remove older
+versions that may exist on your system. To remove old versions of Azure PowerShell from your system,
+see [Uninstall the Azure PowerShell module](uninstall-az-ps.md). For more information about
+MSI-based installations, see [Install Azure PowerShell with an MSI](install-az-ps-msi.md).
 
 ## Use multiple versions of Azure PowerShell
 
-It's possible to install more than one version of Azure PowerShell. To check if you have multiple versions of Azure PowerShell installed, use the following
-command:
+It's possible to install more than one version of Azure PowerShell. To check if you have multiple
+versions of Azure PowerShell installed, use the following command:
 
 ```powershell-interactive
 Get-InstalledModule -Name Az -AllVersions | select Name,Version
 ```
 
 To remove a version of Azure PowerShell, see [Uninstall the Azure PowerShell module](uninstall-az-ps.md).
 
-You can install or load a specific version of the `Az` module by using the `-RequiredVersion` argument:
+If you have more than one version of the module installed, module autoload and `Import-Module` load
+the latest version by default.
+
+You can install or load a specific version of the `Az` module by using the `-RequiredVersion`
+parameter:
 
 ```powershell-interactive
 # Install Az version 0.7.0
```

Comment:
Should we update this to version 3.5.0?

Reply:
Yes, or get rid of the comment or change the comment to "latest"